projects/storyteller/webapp/src/app/core/services/firebase.service.ts

```diff
@@ -34,19 +36,39 @@ export class FirebaseService {
 
           if (firebaseElement === undefined) {
             // Element not found
-          return;
+            return;
           }
           return this.convertFirebaseElementToStory(firebaseElement);
         })));
   }
+  getscreenshotAll(): Observable<Screenshot[]> {
```

Comment:
Please, always use camelCase with javascript and typescript code.
It's worldwide standard.
https://en.wikipedia.org/wiki/Camel_case

Reply:
With camel case it should be: getScreenshotAll()

Reply:
done